src/concrete/ml/sklearn/tree_to_numpy.py

```diff
@@ -183,6 +213,12 @@ def tree_onnx_graph_preprocessing(
         on_error_msg=f"{len(onnx_model.graph.output)} != 2",
     )
 
+    # Check that a XGBoostRegressor onnx graph has the + 0.5 add node.
+    if framework == "xgboost":
+        # Make sure it's a regression and so a single onnx output.
```

Comment:
"and so a single onnx output." weird sentence ?

Reply:
Hmm sounds good to me. What do you suggest?

Reply:
ah ok got it, I would maybe rephrase it it still , like : `Make sure it's a regression model (by checking it has a single output, as mentioned above)` or something like it

Reply:
done